src/py_scripts/fc_fetch_reads.py

```diff
@@ -0,0 +1,3 @@
+from falcon_kit.mains.fetch_reads import main
```

Comment:
There is really no reason to add these py_scripts. Just use the "entry-points" in setup.py.